mantidproject/mantid
# ILL powder few fixes

**Description of work.**

Fixes a couple of problems with ILL powder loader and reducer.

**To test:**

- [  ] 
```
PowderDiffILLDetScanReduction(Run='508093:508095', Output1D = True, OutputWorkspace='list')
PowderDiffILLDetScanReduction(Run='508093-508095', Output1D = True, OutputWorkspace='sum')
comp = CompareWorkspaces(Workspace1='list_1D', Workspace2='sum_1D')
assert(comp.Result)
```

- [  ]
```
PowderDiffILLDetEffCorr(CalibrationRun='967076.nxs', OutputWorkspace='calib', OutputResponseWorkspace='response')
```
   Plot `response`, make sure that the scattering angle 0 is within the beam stop range (the range with almost no intensity).

Data is in system tests under ILL/D2B and ILL/D20 respectively.

The rest are captured by the tests.


Fixes #24851 . 


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
